website/source/api/auth/aws/index.html.md

```diff
@@ -548,52 +550,63 @@ inferencing configuration of that role.
   "ec2"). Only those bindings applicable to the auth type chosen will be allowed
   to be configured on the role.
 - `bound_ami_id` `(string: "")` - If set, defines a constraint on the EC2
```

Comment:
`s/(string: "")/(list: [])` for all the fields that are upgraded to TypeStringCommaSlice

Reply:
done (FWIW, I just copied what was done for the PKI secret backend, e..g, http://localhost:4567/api/secret/pki/index.html#ou-1).